docs/devGuide/devGuide.md

```diff
@@ -136,20 +136,41 @@ $ npm run test
 For Windows users:
 
 ```
-$ npm run testwin
+> npm run testwin
 ```
 
 #### Updating tests
 
 When adding new features, updating existing features or fixing bugs, you should update the expected site to reflect the changes.
 
+You may use the following script to update the expected sites:
+
+For Unix:
+
+```
+$ npm run updatetest
+```
+
+For Windows users:
+
+```
+> npm run updatetestwin
+```
+<box type="warning">
+  You should always check that the output generated is correct before committing your changes to the expected site.
```

Comment:
Let's standardize on "expected files" and "test sites"? :D

Which means something like: `You should always check that the generated output is correct before committing any changes to the test sites.`